integreat_cms/cms/views/pois/poi_context_mixin.py

```diff
@@ -72,6 +72,7 @@ def get_context_data(self, **kwargs):
                     "Opening time is identical with the closing time of the previous slot"
                 ),
             },
+            "canChangeEvent": self.request.user.has_perm("cms.change_event"),
```

Comment:
```suggestion
            "canChangeLocation": self.request.user.has_perm("cms.change_poi"),
```
I think we should only check for the poi permission when editing pois, right?
This also impacts all other occurrences of this string.